backend/pom.xml

```diff
@@ -164,6 +165,31 @@
                 <artifactId>clover-maven-plugin</artifactId>
```

Comment:
I think we can remove openclover since we are using jacoco in sonarcloud and do not need openclover to generate test coverage reports. Minor detail, it can be removed in another PR too

Reply:
I think we should remove it as it falls under refactoring.